com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -526,7 +532,7 @@ bool RequiresIntermediateColorTexture(ref CameraData cameraData)
             int msaaSamples = cameraTargetDescriptor.msaaSamples;
             bool isScaledRender = !Mathf.Approximately(cameraData.renderScale, 1.0f);
             bool isCompatibleBackbufferTextureDimension = cameraTargetDescriptor.dimension == TextureDimension.Tex2D;
-            bool requiresExplicitMsaaResolve = msaaSamples > 1 && !SystemInfo.supportsMultisampleAutoResolve;
+            bool requiresExplicitMsaaResolve = msaaSamples > 1 && PlatformRequiresExplicitAutoResolve();
```

Comment:
what about deferred renderer?

Reply:
I did a search for usages of SystemInfo.supportsMultisampleAutoResolve. Both the deferred and the 2D renderer use it to set the Depth texture BindMS flag:

depthDescriptor.bindMS = msaaSamples > 1 && !SystemInfo.supportsMultisampleAutoResolve && (SystemInfo.supportsMultisampledTextures != 0);

so it is not the same issue as the one I am fixing here, but it should probably be investigated as the autoResolve flag couldn't be the ideal check on some platforms?